src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -47,14 +47,14 @@ tvt satcheck_minisat2_baset::l_get(literalt a) const
 
   tvt result;
 
-  if(a.var_no()>=(unsigned)solver->model.size())
+  if(a.var_no()>=(unsigned)solver_->model.size())
```

Comment:
WHY?

Reply:
GitHub, Vim, Emacs, Visual Studio, QT, VS Code and AFAIK Clion do not highlight member variables or member functions. C++ does not enforce `this->`. Therefore it's often difficult to discern whether a variable is a member or local. Every time such variable is encountered, the reader has to spend considerable amount of time going through the entire file looking for globals, statics, members and/or child classes.

We've spent months arguing about this and finally postfix underscore was agreed upon to signify a member variable. On top of that `solver_` was already used as an argument name.

Reply:
Where is this documented in the coding standard?

At the same time: this PR is already way too large, why include such style changes in there?